src/phases/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 
 # Install all the phase config snippets.
 install(DIRECTORY .
-    DESTINATION etc/genny/phase_configs
+    DESTINATION etc/genny/phases
```

Comment:
fixing this bug from a previous ticket.

Reply:
(separately I was thinking that `phases` may want to be a subdir of `workloads`...I added to the ideas doc)